app/models/spree/line_item_decorator.rb

```diff
@@ -109,7 +109,7 @@ def price_with_adjustments
 
     line_item_adjustments = OrderAdjustmentsFetcher.new(order).line_item_adjustments(self)
 
-    (price + line_item_adjustments.sum(&:amount) / quantity).round(2)
+    (price + line_item_adjustments.to_a.sum(&:amount) / quantity).round(2)
```

Comment:
Isn't `amount` an attribute on adjustment records? Can this move to #5921 as `line_item_adjustments.sum(:amount)`?